init-tools.ps1

```diff
@@ -57,6 +57,61 @@ if (!(Test-Path "$BUILD_TOOLS_PACKAGE_PATH\init-tools.cmd"))
     exit 1
 }
 
+# Bring down the CLI for build tools
+$DOTNET_PATH=$BUILD_TOOLS_PATH + "\dotnetcli"
+
+Write-Host "Installing Build Tools CLI version..."
+if (!(Test-Path "$DOTNET_PATH"))
+{
+     mkdir "$DOTNET_PATH"
+}
+
+$DOTNET_VERSION = Get-Content "$RepoRoot\BuildToolsCLIVersion.txt"
+$DOTNET_ZIP_NAME="dotnet-dev-win-x64." + $DOTNET_VERSION + ".zip"
```

Comment:
Either option has stale code; the code you linked to relies on us having preview2 support in the obtain script in rel-1.0.0...as seen with the current install script, we cannot rely on that assumption. 

The correct fix is having the obtain script work with pre-preview3 bits; however, until that comes online, the easiest path forward is to do what corefx and coreclr and every other repo does in the init script, which is `wget` the bits. There is a dependency on the location of preview2 set forth by the other repos; once they set the precedence for moving to preview3 (or preview3+), we can use the obtains script as is directed. 

Keeping this as-is allows us to stay in step with the other repos until either a) the obtain script works with pre-preview3 or b) build tools upgrades to preview3+